### PR TITLE
Saint Roumain Militia Content Expansion Part 2: New Jobs! 

### PR DESCRIPTION
**Pull request by Imaginos16**
## About The Pull Request
THE LONG AWAITED SEQUEL!

This PR adds a ton of things, but let's start with the most important changes:

image

Not only has every job has been tweaked and resprited visually, but you might've noticed at least three unfamiliar faces to the roster of the Saint Roumain's Militia! This is because now, there are new jobs for people to enjoy!

These include:

- The Flamebearer, a member of the ecclesiastical sect of the Church of Saint Roumain, focused on preaching the word of Saint Roumain, the Ashen Huntsman, through the holy text of the Liber Incendium!

- The Machinist, a dedicated hunter focused on engineering prowess. Most workers of Hunter's Pride incidentally are Machinists by trade back in Illestren!

- The Hunter Colligne, effectively a secondary rank to the Montagne. They are trainee Hunter Montagnes effectively, learning the art of hunting and commandeering to eventually lead their own cadre of Hunters as a Montagne.

All jobs within the SRM also have unique job icons exclusive to them. Hope you enjoy it!

The SRM now boast an exclusive hardsuit for themselves, based on knight's armor!

image

This PR tweaks some minor stuff aboard the Elder-class, specifically to accomodate the new jobs. A Shadow slot has been killed in exchange for one Colligne, one Machinist and one Flamebearer slot.

And finally, this PR does the absolutely random thing of updating the ID-less icon and its background for no other reason other than I could.
image

## Why It's Good For The Game
Pleade join my shipse I begge...

## Changelog

:cl: PositiveEntropy
add: The Hunters of Illestren have now allowed for Hunter Collignes, Machinists and Flamebearers to now roam the Frontier aboard their humble fleet!
imageadd: The Saint Roumain's Militia has been fully resprited!
imageadd: All SRM jobs now have unique HUD icons!
imageadd: The Unknown Job HUD icon has been resprited!
/:cl:

### CODE REVIEW

**Comment by generalthrax** [comment]
The Fire Within

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -444,27 +444,6 @@
 	combat_slowdown = 0.5
 	jetpack = null
 
-//Hardliner Syndie suit
```
Comment (SomeguyManperson) [inline code/modules/clothing/spacesuits/hardsuit.dm:447]: ?

Comment (Imaginos16) [reply code/modules/clothing/spacesuits/hardsuit.dm:447]: Duplicate path I killed.

File: code/modules/clothing/factions/srm.dm [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+	name = "saint-roumain machinist leather vest"
+	desc = "A modified Roumain leather duster with its large flaps and sleeves cut off to provide extra mobility when maintaining weapons and vessels belonging to the Church of Saint Roumain. Its specialty treatment grants it better protection against acid and fire."
+	icon = 'icons/obj/clothing/faction/srm/suits.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/srm/suits.dmi'
+	icon_state = "armor_rouma_machinist"
+	item_state = "rouma_coat"
+	armor = list("melee" = 30, "bullet" = 20, "laser" = 20, "energy" = 40, "bomb" = 35, "bio" = 0, "rad" = 0, "fire" = 60, "acid" = 60)
+
+/obj/item/clothing/suit/armor/roumain/flamebearer
+	name = "saint-roumain flambearer robes"
```
Comment (Apogee-dev) [inline code/modules/clothing/factions/srm.dm]: Typo here ("flambearer"). Should be "flamebearer"

File: code/modules/clothing/factions/srm.dm [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+	desc = "A set of ashy-grey robes made from hard leather, adorned with gold trims. Its rough finish after a near-char and application of aromatics is heavily favored for the ecclesiastical sect of the Church of Saint Roumain, a living reminder of the Ashen Huntsman himself."
```
Comment (Apogee-dev) [inline code/modules/clothing/factions/srm.dm:54]: "A set of ash-grey robes made from hard leather and adorned with gold trim and impregnated with aromatic oils. The rough, nearly-charred finish is a favorite of the ecclesiastical sect of the Church of Saint Roumain, a constant reminder of the Ashen Huntsman himself."

File: code/modules/clothing/factions/srm.dm [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+	icon_state = "armor_rouma_flamebearer"
+
+/obj/item/clothing/suit/armor/roumain/colligne
+	name = "saint-roumain colligne coat"
+	desc = "A well-maintained hard leather coat typically worn to denote the rank of Colligne, a trainee Hunter Montagne. It is treated with bullet-resistant materials, and lined with the dark fur of Illestrenin dire wolves."
```
Comment (Apogee-dev) [inline code/modules/clothing/factions/srm.dm]: Typo here ("Illestrenin"). Should be "Illestrian"

File: code/modules/clothing/factions/srm.dm [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+	cold_protection = CHEST|GROIN|LEGS|ARMS
+	heat_protection = CHEST|GROIN|LEGS|ARMS
+
+///////////////
+//Spacesuits//
+//////////////
+
+/obj/item/clothing/head/helmet/space/hardsuit/solgov/roumain
+	name = "\improper roumain hardsuit helmet"
+	desc = "An armor-plated knight's helmet, the pointed design of it reminiscent of iconography depicting the great Ashen Huntsman."
```
Comment (Apogee-dev) [inline code/modules/clothing/factions/srm.dm]: "An armored helmet with an unusual design that recalls both pre-industrial Solarian armor and iconography depicting the Ashen Huntsman. Though hand-made, it is surprisingly quite spaceworthy."

File: code/modules/clothing/factions/srm.dm (hunk 2 above)
Comment (thgvr) [inline code/modules/clothing/factions/srm.dm]:
```suggestion
	name = "saint-roumain flamebearer robes"
```

File: code/modules/clothing/factions/srm.dm [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+	icon = 'icons/obj/clothing/faction/srm/head.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/srm/head.dmi'
+	icon_state = "hardsuit0-roumain"
+	item_state = "hardsuit0-roumain"
+	hardsuit_type = "roumain"
+	worn_y_offset = 4
+
+/obj/item/clothing/suit/space/hardsuit/solgov/roumain
+	name = "\improper roumain hardsuit"
+	desc = "A modified knight's plate armor turned into a space-worthy hardsuit. A powered exoskeleton hidden behind the plates help grant it slight maneuverability."
```
Comment (Apogee-dev) [inline code/modules/clothing/factions/srm.dm]: Suggested rewrite to drive home the artisanal nature of most SRM gear and make Hunter's Pride more than a little more explicitly badass: 
"A hand-crafted suit of armor either modified from a set of normal plate armor or designed to resemble one. A powered exoskeleton has been cleverly integrated into the design and, surprisingly, it is completely vacuum-proof. Suits like this are a testament to what the master craftsmen of Hunter's Pride are capable of."

**Review by Apogee-dev — DISMISSED** [review]

**Pull request merged**